src/app/child-dev-project/notes/model/note.ts

```diff
@@ -15,26 +15,38 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { MeetingNoteAttendance } from "../meeting-note-attendance";
 import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
 import { Entity } from "../../../core/entity/entity";
 import { DatabaseField } from "../../../core/entity/database-field.decorator";
 import { WarningLevel, WarningLevelColor } from "../../warning-level";
 import { InteractionType } from "../note-config-loader/note-config.interface";
+import { EventAttendance } from "../../attendance/model/event-attendance";
+import { AttendanceStatus } from "../../attendance/model/attendance-status";
+import { v4 as uuid } from "uuid";
 
 @DatabaseEntity("Note")
 export class Note extends Entity {
+  static create(date: Date, subject: string = ""): Note {
```

Comment:
I started this pattern here to make it easier creating instances with some preset values (otherwise you have to first create a new instance and then assigned each property in a separate line). Changing the constructor turned out to be problematic because EntityMapper expects the constructor to have exactly one parameter always.

I think we could introduce this for other Entity classes whenever needed. What's your opinion in general about such a factory method?

Reply:
I think it would be very useful, especially for setting the entity id